src/main/java/core/ChoiceInputValidator.java

```diff
@@ -17,11 +17,20 @@ public ChoiceInputValidator(List<String> allowedInputs) {
     }
 
     /**
-     * @param input the user input to validate
-     * @return whether the given input is valid
+     * @param input the user input to parse and validate
+     * @return null if the given input is not valid, otherwise the parsed input
      */
     @Override
-    public boolean validateInput(String input) {
-        return allowedInputs.contains(input);
+    public String parseAndValidate(String input) {
+        String parsed = input.toUpperCase();
+        if (parsed == "EXIT" || parsed == "PAUSE") {
```

Comment:
This part should be handled in the inputHandler - otherwise wed need this code block in every single inputValidator